### PR TITLE
Add Jinan to Contributor List

### PR DESCRIPTION
**Pull request by DeeperMind**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/181)

### CODE REVIEW

File: members.yaml [hunk 1]
```diff
@@ -72,6 +72,13 @@
   lastName: Dutta
   affiliation: Cisco
   email: dedutta@cisco.com
+  
+- github: deepermind
+  slack: jinanzhou
+  firstName: Jinan
+  lastName: Zhou
+  affiliation: Cisco
+  emial: jinazhou@cisco.com
```
Comment (abhi-g) [inline members.yaml]: typo: email

**Comment by abhi-g** [comment]
@DeeperMind Please sign the CLA. See link above added by googlebot.

File: members.yaml (hunk 1 above)
Comment (DeeperMind) [reply members.yaml]: fixed. thx!

**Pull request merged**